ksqldb-execution/src/main/java/io/confluent/ksql/execution/windows/HoppingWindowExpression.java

```diff
@@ -80,7 +92,7 @@ public String toString() {
         + "ADVANCE BY " + advanceBy
         + retention.map(w -> " , RETENTION " + w).orElse("")
         + gracePeriod.map(g -> " , GRACE PERIOD " + g).orElse("")
-        + " ) ";
+        + " ) " + emitStrategy.map(Enum::toString).orElse("");
```

Comment:
This is a nasty hack -- did not find a better way to preserve the information.

Reply:
For other readers: this "hack" is being added so that deserialization works properly here: https://github.com/confluentinc/ksql/pull/9141#discussion_r879937488